<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>YouTubeDlGui</class>
 <widget class="QMainWindow" name="YouTubeDlGui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>718</width>
    <height>429</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>YouTube DL GUI</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../res/resources.qrc">
    <normaloff>:/images/youtube-16.png</normaloff>:/images/youtube-16.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="5">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="btnUp">
        <property name="text">
         <string>&amp;Up</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnDown">
        <property name="text">
         <string>&amp;Down</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnRemove">
        <property name="text">
         <string>&amp;Remove</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btnDownload">
        <property name="text">
         <string>Down&amp;load</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="0" colspan="5">
     <widget class="QTableWidget" name="tblQueue">
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="2" colspan="2">
     <widget class="QComboBox" name="cmbFormat"/>
    </item>
    <item row="0" column="5">
     <widget class="QPushButton" name="btnAdd">
      <property name="text">
       <string>&amp;Add</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="txtUrl">
      <property name="placeholderText">
       <string>YouTube Video URL</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Maximum</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>15</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>718</width>
     <height>29</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionPreferences"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="../res/resources.qrc">
     <normaloff>:/images/quit-16.png</normaloff>:/images/quit-16.png</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="icon">
    <iconset resource="../res/resources.qrc">
     <normaloff>:/images/preferences-16.png</normaloff>:/images/preferences-16.png</iconset>
   </property>
   <property name="text">
    <string>Preferences</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="../res/resources.qrc">
     <normaloff>:/images/about-16.png</normaloff>:/images/about-16.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../res/resources.qrc"/>
 </resources>
 <connections/>
</ui>
